WebKitBrowser/JSPP.config

```diff
@@ -124,6 +124,10 @@ map()
             kv(name "XDG_DATA_HOME")
             kv(value "/opt/QT/home")
         end()
+        map()
+            kv(name "PROCESS_NAME")
+            kv(value "JSPP-WPEWebProcess")
```

Comment:
Let's name it JSPPWebProcess, just to keep name short.

Reply:
Changed.